wakuv2/nwaku_test.go

```diff
@@ -161,80 +154,87 @@ func parseNodes(rec []string) []*enode.Node {
 // Using Docker:
 //
 //	IP_ADDRESS=$(hostname -I | awk '{print $1}');
-//	docker run \
-//	 -p 60000:60000/tcp -p 9000:9000/udp -p 8645:8645/tcp harbor.status.im/wakuorg/nwaku:v0.31.0 \
-//	 --tcp-port=60000 --discv5-discovery=true --cluster-id=16 --pubsub-topic=/waku/2/rs/16/32 --pubsub-topic=/waku/2/rs/16/64 \
-//	 --nat=extip:${IP_ADDRESS} --discv5-discovery --discv5-udp-port=9000 --rest-address=0.0.0.0 --store
+// 	docker run \
+// 	-p 61000:61000/tcp -p 8000:8000/udp -p 8646:8646/tcp harbor.status.im/wakuorg/nwaku:v0.33.0 \
+// 	--discv5-discovery=true --cluster-id=16 --log-level=DEBUG \
+// 	--nat=extip:${IP_ADDRESS} --discv5-udp-port=8000 --rest-address=0.0.0.0 --store --rest-port=8646 \
 
 func TestBasicWakuV2(t *testing.T) {
-	nwakuInfo, err := GetNwakuInfo(nil, nil)
-	require.NoError(t, err)
-
-	// Creating a fake DNS Discovery ENRTree
-	tree, url := makeTestTree("n", parseNodes([]string{nwakuInfo.EnrUri}), nil)
-	enrTreeAddress := url
-	envEnrTreeAddress := os.Getenv("ENRTREE_ADDRESS")
-	if envEnrTreeAddress != "" {
-		enrTreeAddress = envEnrTreeAddress
+	extNodeRestPort := 8646
+	storeNodeInfo, err := GetNwakuInfo(nil, &extNodeRestPort)
+	require.NoError(t, err)
+	
+	nwakuConfig := WakuConfig{
+		Port:        30303,
+		NodeKey:     "11d0dcea28e86f81937a3bd1163473c7fbc0a0db54fd72914849bc47bdf78710",
+		EnableRelay: true,
+		LogLevel:    "DEBUG",
+		DnsDiscoveryUrl: "enrtree://AMOJVZX4V6EXP7NTJPMAYJYST2QP6AJXYW76IU6VGJS7UVSNDYZG4@boot.prod.status.nodes.status.im",
```

Comment:
This while it will work, it has the inconvenient of polluting the bootnodes with nodes that will only live when we run the tests (hence why the fake enr tree was being created)

Reply:
Makes sense 100%! As we discussed, I created the following issue to track this improvement https://github.com/waku-org/nwaku/issues/3119

Thanks so much!